lib/jest-helpers.js

```diff
@@ -21,7 +20,7 @@ function render (componentName, params) {
 
 function getExamples (componentName) {
   const file = fs.readFileSync(
-    path.join(COMPONENT_DIR, `${componentName}/${componentName}.yaml`),
+    path.join(configPaths.components, `${componentName}/${componentName}.yaml`),
```

Comment:
Suggest letting path.join work it's magic here:

```
path.join(configPaths.components, componentName, `${componentName}.yaml`)
```

Reply:
Updated 👍